src/ripple/app/tx/impl/SetHook.cpp

```diff
@@ -452,6 +455,13 @@ SetHook::validateHookSetEntry(SetHookCtx& ctx, STObject const& hookSetObj)
                 return false;
             }
 
+            // validate sfHookCanEmit
+            // HookCanEmit field is an optional field for backward compatibility
+            if (!hookSetObj.isFieldPresent(sfHookCanEmit))
+            {
+                // pass
```

Comment:
What's this block for?

Reply:
I had added this block for readability to match the processing of other existing fields.